src/d3a/setup/jira/d3asim_1531.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
@spyrostz @faizan2590 I kept this here for now. I am not sure whether an integrationtest that checks for trades happening in the only root market is an overkill or not. What do you think?

Reply:
I would actually suggest to add an integration test that trades indeed happen. The reason I mention that is because in this case you have only one market.